packages/ai-ide/src/browser/style/index.css

```diff
@@ -348,3 +348,104 @@
   font-weight: bolder;
   font-style: normal;
 }
+
+/* Token Usage Configuration Styles */
+.token-usage-configuration-container {
+  padding: 16px;
+}
+
+.token-usage-configuration-title {
+  margin: 0 0 16px 0;
+  border-bottom: 1px solid var(--theia-panelTitle-activeBorder);
```

Comment:
I think we shouldn't use the active border here. This color is somewhat reserved for components that have focus and are active. Here we use this color all the time.